Sources/SourceControl/GitRepository.swift

```diff
@@ -16,13 +16,13 @@ import func POSIX.getenv
 import enum POSIX.Error
 
 enum GitRepositoryProviderError: Swift.Error {
-    case gitCloneFailure(url: String, path: AbsolutePath)
+    case gitCloneFailure(url: String, path: AbsolutePath, errorOutput: String)
```

Comment:
Think of it as API and clients, this module is provided and API and Commands is a client which handles that error.
Command is basically a high level client which contains the cli tools: swift-{build, test, package}

You can ignore Get, it's dead code now.